cv_checker_backend/cv_checker_backend/core/jwt.py

```
@@ -0,0 +1,53 @@
+from jose import jwt, JWTError
+from  typing import Dict, Optional
+from fastapi.security import OAuth2PasswordBearer
```

Ruff (0.8.2)

`fastapi.security.OAuth2PasswordBearer` imported but unused

Remove unused import: `fastapi.security.OAuth2PasswordBearer`

(F401)